runtime/makelib/targets.mk.linux.inc.ftl

```diff
@@ -273,7 +273,7 @@ CPPFLAGS += -DLINUX -D_REENTRANT
 </#if>
 
 <#-- Add Position Independent compile flag -->
-<#if uma.spec.processor.amd64 || uma.spec.processor.arm || uma.spec.processor.s390 || uma.spec.processor.riscv64>
+<#if uma.spec.processor.amd64 || uma.spec.processor.arm || uma.spec.processor.aarch64 || uma.spec.processor.s390 || uma.spec.processor.riscv64>
```

Comment:
My (limited) understanding of the `-fPIC` option is that it can produce larger and slower code than `-fpic`, albeit with less target dependence.  Given the interest in keeping the AArch64 JIT shared library as small as possible, is there a footprint difference observed between the two link options that needs to be taken into consideration here?

Reply:
The JIT and GC binaries compiled with `-fPIC` are smaller than those with `-fpic`, while some other files are bigger with `-fPIC`.
`-fPIC` makes the total footprint smaller.

with `-fPIC`:
```
 12614192  2020-12-07 21:30   jdk/lib/compressedrefs/libj9jit29.so
  3249176  2020-12-07 21:30   jdk/lib/compressedrefs/libj9gc29.so

  1570584  2020-12-07 21:30   jdk/lib/compressedrefs/libj9vm29.so
   657800  2020-12-07 21:30   jdk/lib/compressedrefs/libj9shr29.so

(total)
297659854                     774 files
```

with `-fpic`:
```
 12626872  2020-12-07 20:38   jdk/lib/compressedrefs/libj9jit29.so
  3253512  2020-12-07 20:38   jdk/lib/compressedrefs/libj9gc29.so

  1566416  2020-12-07 20:38   jdk/lib/compressedrefs/libj9vm29.so
   648888  2020-12-07 20:38   jdk/lib/compressedrefs/libj9shr29.so

(total)
297685871                     774 files
```

Reply:
Thanks for measuring real data.  I'll have to dig into the real differences between those two options someday.  Change looks good.